# 12 异常

#####异常处理程序
抛出异常
监控区域
- 终止模型
- 恢复模型

fillInstackTrace : 把当前调用栈信息填入原来的对象

#####异常链
cause initcause

runtimeException 不受检查异常，流程错误，被捕获

#####finally
异常丢失
finally 覆盖
finally 返回（return)

文件关闭
catch
finally

把受检查的异常转换为不受检查，包装到RuntimeException中，使用getCause处理冒泡的异常
